pkg/addon/templateagent/registration.go

```diff
@@ -25,13 +27,29 @@ import (
 )
 
 const (
-	TLSCACert = "ca.crt"
-	TLSCAKey  = "ca.key"
 	// AddonTemplateLabelKey is the label key to set addon template name. It is to set the resources on the hub relating
 	// to an addon template
 	AddonTemplateLabelKey = "open-cluster-management.io/addon-template-name"
 )
 
+var (
+	podNamespace = ""
+)
+
+func AddonManagerNamespace() string {
```

Comment:
I think this should be able to be passed from outside? The controllerCtx have this namespace

Reply:
This will be used by the `TemplateCSRSignFunc` which does have the controllerCtx . And I think using downward api to get the pod namespace is easier.